qemu/tests/chardev_serial_login.py

```diff
@@ -10,8 +10,8 @@
 @error_context.context_aware
 def run(test, params, env):
     """
-    Verify the login function of pci-serial (RHEL and x86 only):
-    1) Start guest with pci-serial with backend
+    Verify the login function of chardev-serial (RHEL only):
+    1) Start guest with chardev-serial with backend
```

Comment:
Can we update "chardev-serial"  to "chardev_spapr_vty_login"  since it is much clearer

Reply:
@MiriamDeng Hi This is a misunderstanding, in order to prevent code redundancy, 'py' files we share with the x86.
So this place can't just use spapr_vty to illustrate.